crates/oxc_transformer/src/lib.rs

```diff
@@ -144,4 +144,14 @@ impl<'a> VisitMut<'a> for Transformer<'a> {
             self.visit_class_element(class_element);
         });
     }
+
+    fn visit_formal_parameters(&mut self, params: &mut FormalParameters<'a>) {
```

Comment:
Does this mean that we need to place a lot of AST visitor functions?
TypeScript AST are in many syntactic positions, and they produce a lot of extra statements.

However, when these AST visitor functions need to cooperate with each other in context, this will become very complicated.

Reply:
It's a clever method.
So, I supposed there should be some methods, such as getInnerExpr, to get the JavaScript context AST that is wrapped by the TypeScript AST. Right?